src/pages/pick-a-rice/components/MenuScreen.module.css

```
@@ -0,0 +1,485 @@
+}
+
+.letterBubbleHide {
+  animation: bubbleHide 1ms steps(1, end) forwards;
+  animation-delay: var(--bubble-delay);
+}
+
+.letterBubbleHidden {
+  visibility: hidden;
+}
+
+@keyframes bubbleShow {
```

Stylelint (17.9.0)

[error] Expected keyframe name "bubbleShow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)